### ISSUE TITLE
Duskbringer stun-cleansing glyph is not purging Fortune Seeker's "Boundless Strike" stun

### ISSUE THREAD

**Issue by DeckardCainRPC** [ChalkyBrush/roshpit-bug-tracker#905]

**Comment by redledcat** [comment]
Well, problem with both of functions CleanseStuns and CleanseSilences is each stunning and silencing effect must be added to the list.
So, many abilities which do not use standart modifiers for stuns and slences wouldn't be cleased, such as several abilities added in Winterblight part 1, that curse of azalea silens, stuns of giant svens, etc.
(well, svens add root and disarm + stun, maybe that's why)
such ability as boundless strike, copied from Dota 2 itself, are not added to that function, so they become such a random pain.
Tho curse of azalea meant to be as painful as it can be.

**Comment by redledcat** [comment]
maybe boundless strike uses bash instead of modifier_stunned, no idea

**Comment by DeckardCainRPC** [comment]
yeah i know that each effect has to be manually entered into a whitelist

**Comment by redledcat** [comment]
problem is I don't know how stunning modifier named of boundless strike
wiki says that is one of abilities which is neither modifier_stunned nor bash, but unique one
https://dota2.gamepedia.com/Stun
but I have no idea and skills how to extract proper name of that modifier to add this to the cleance list

**Comment by redledcat** [comment]
at least it explained me why cleansing mechanics have so many lacks when it meets dota 2 spells
